var/spack/repos/builtin/packages/py-configargparse/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyConfigargparse(PythonPackage):
+    """A drop-in replacement for argparse that allows options to also be set via config files and/or environment variables.
```

Comment:
multi-line to avoid [flake8](https://spack.readthedocs.io/en/latest/contribution_guide.html?highlight=80%20char#flake8-tests) error?